homeassistant/components/cover/group.py

```diff
@@ -0,0 +1,277 @@
+        self._cover_position = 100
+        self._tilt_position = None
+        self._supported_features = 0
+        self._assumed_state = True
+
+        self._entities = entities
+        self._covers = {KEY_OPEN_CLOSE: set(), KEY_STOP: set(),
+                        KEY_POSITION: set()}
+        self._tilts = {KEY_OPEN_CLOSE: set(), KEY_STOP: set(),
+                       KEY_POSITION: set()}
+
+    @callback
+    def update_supported_features(self, entity_id, old_state, new_state,
+                                  update_state=True):
+        """Update dictionaries with supported features."""
+        if new_state is None:
+            for values in self._covers.values():
+                values.discard(entity_id)
+            for values in self._tilts.values():
+                values.discard(entity_id)
+        else:
+            features = new_state.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
+
+            # pylint: disable=expression-not-assigned
+            self._covers[KEY_OPEN_CLOSE].add(entity_id) \
+                if features & (SUPPORT_OPEN | SUPPORT_CLOSE) \
+                else self._covers[KEY_OPEN_CLOSE].discard(entity_id)
+
+            # pylint: disable=expression-not-assigned
+            self._covers[KEY_STOP].add(entity_id) \
+                if features & (SUPPORT_STOP) \
+                else self._covers[KEY_STOP].discard(entity_id)
+
+            # pylint: disable=expression-not-assigned
+            self._covers[KEY_POSITION].add(entity_id) \
+                if features & (SUPPORT_SET_POSITION) \
+                else self._covers[KEY_POSITION].discard(entity_id)
+
+            # pylint: disable=expression-not-assigned
+            self._tilts[KEY_OPEN_CLOSE].add(entity_id) \
+                if features & (SUPPORT_OPEN_TILT | SUPPORT_CLOSE_TILT) \
+                else self._tilts[KEY_OPEN_CLOSE].discard(entity_id)
+
+            # pylint: disable=expression-not-assigned
+            self._tilts[KEY_STOP].add(entity_id) \
+                if features & (SUPPORT_STOP_TILT) \
+                else self._tilts[KEY_STOP].discard(entity_id)
+
+            # pylint: disable=expression-not-assigned
+            self._tilts[KEY_POSITION].add(entity_id) \
+                if features & (SUPPORT_SET_TILT_POSITION) \
+                else self._tilts[KEY_POSITION].discard(entity_id)
+
+        tilt = False
+        for values in self._tilts.values():
+            if not values:
+                break
+        else:
+            tilt = True
+        self._tilt = tilt
+
+        if update_state:
+            self.async_schedule_update_ha_state(True)
+
+    async def async_added_to_hass(self):
+        """Register listeners."""
+        for entity_id in self._entities:
+            new_state = self.hass.states.get(entity_id)
+            self.update_supported_features(entity_id, None, new_state,
+                                           update_state=False)
+        self.async_schedule_update_ha_state(True)
```

Comment:
The core already calls `self.async_schedule_update_ha_state` for you (https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity_platform.py#L257-L260)

Just call `await self.async_update()` (see https://github.com/home-assistant/home-assistant/pull/12962#discussion_r173324993)